client/README.md

```diff
@@ -96,16 +96,15 @@ import (
     "time"
```

Comment:
Small typo that can be fixed on line 23 [here](https://github.com/Shopify/toxiproxy/pull/607/files#diff-c4577d0b3a9c55c4e75639bb45f3dd659e83bc31b7a30943fda6ef84f7cef47aR23)